eclair-core/src/main/scala/fr/acinq/eclair/channel/Helpers.scala

```diff
@@ -511,16 +492,19 @@ object Helpers {
       }
       val (closingTx, closingSigned) = makeClosingTx(keyManager, commitments, localScriptPubkey, remoteScriptPubkey, remoteClosingFee)
       val signedClosingTx = Transactions.addSigs(closingTx, keyManager.fundingPublicKey(commitments.localParams.fundingKeyPath).publicKey, remoteParams.fundingPubKey, closingSigned.signature, remoteClosingSig)
-      Transactions.checkSpendable(signedClosingTx).map(x => signedClosingTx.tx).recover { case _ => throw InvalidCloseSignature(commitments.channelId, signedClosingTx.tx) }
+      Transactions.checkSpendable(signedClosingTx).map(_ => signedClosingTx.tx).recover { case _ => throw InvalidCloseSignature(commitments.channelId, signedClosingTx.tx) }
```

Comment:
Alternative without `throw`:

```suggestion
      Transactions
        .checkSpendable(signedClosingTx)
        .transform(_ => Success(signedClosingTx.tx), _ => Failure(InvalidCloseSignature(commitments.channelId, signedClosingTx.tx)))
```

Reply:
`match`/`case` would work too.

Reply:
Done in https://github.com/ACINQ/eclair/pull/1366/commits/46e0a10dca00a7cc99bc633ed27f556edaa1f924